code-ready/crc
# Misc Proxy fixes

- No need to remove CNI network definition forcefully
- Rename dnsmasq container to dnsmasq-crc
- Create systemd unit file for dnsmasq container

How to test this PR:
- Download the latest bundle 4.8.11
- Set the proxy setting 
- Set proxy ca file
- Run crc without this patch and it will fails to stabilize the cluster after the failure you can try to export the kubeconfig file from `~/.crc/machines/crc/kubeconfig` and check `oc get nodes` which will be `Nonscheduled` because of dnsmasq container is not running (can be checked after ssh to the vm).
- Run crc with this patch and it succeed

you can use virt-viewer or virt-manager to open the crc instance to keep an eye when it rebooted by MCO.
